internal/rules/policies/kubernetes/policies/advanced/default_namespace_should_not_be_used.rego

```diff
@@ -0,0 +1,33 @@
+# METADATA
+# title: "The default namespace should not be used"
+# description: "ensure that default namespace should not be used"
+# scope: package
+# schemas:
+# - input: schema.input
```

Comment:
there's a dedicated schema for kubernetes (and for RBAC, for the other check) in defsec. Since I think other checks don't make use if it, ok to keep it like this for now. and we'l maybe address it later. Just noting this FYI.